packages/expo-cli/src/commands/codesigning/configureCodeSigningAsync.ts

```diff
@@ -31,16 +31,26 @@ export async function actionAsync(projectRoot: string, { input }: Options) {
   validateSelfSignedCertificate(certificate, keyPair);
 
   const { exp } = getConfig(projectRoot, { skipSDKVersionRequirement: true });
+
+  const fields: ExpoConfig['updates'] = {
+    codeSigningCertificate: `./${input}/certificate.pem`,
```

Comment:
IMO it might be better to normalize `input` by doing
```suggestion
    codeSigningCertificate: `${path.relative(projectRoot, inputDir)}/certificate.pem`,
```